exporter/main.go

```diff
@@ -32,7 +32,7 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
-		go newMetricsExporter(
+		newMetricsExporter(
```

Comment:
If you remove the `go` keyword here, you won't move on to serving metrics until you're done collecting them-- under normal circumstances, never.